src/tasks/tasks.service.spec.ts

```diff
@@ -15,4 +25,32 @@ describe('TasksService', () => {
   it('should be defined', () => {
     expect(service).toBeDefined();
```

Comment:
Why we are always checking for the services to be defined?

Reply:
this is the default test generated when we create a test file, so i thought to leave it as it is.

Reply:
@haziqAhmed92 what are your views on this?

Reply:
it will always be defined, cause you are importing it, otherwise the project would not run. 
please remove them as it is of no importance

Reply:
Removed.